tests/kafkatest/services/connect.py

```diff
@@ -76,6 +78,41 @@ def set_configs(self, config_template_func, connector_config_templates=None):
         self.config_template_func = config_template_func
         self.connector_config_templates = connector_config_templates
 
+    def listening(self, node, port=8083):
```

Comment:
Is this an action or a query function?

Reply:
I'll add a comment here. But it's equivalent, to the one in zookeeper.py